.github/workflows/continuous-deployment.yml

```diff
@@ -209,12 +209,24 @@ jobs:
           setupGitUser: false
           title: "docs(release): design system packages"
           publish: "pnpm run publish"
+          version: "pnpm run version"
 
-      - name: "Continuous Deployment: publish to npm"
+      # Publish canary releases only if the packages weren't published already
+      - name: Publishing canary releases to npm
+        if: steps.changeset.outputs.published != 'true'
+        run: |
+          git checkout main
+          pnpm run version:canary
+          pnpm run publish:canary
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: |
-          pnpm config set "//registry.npmjs.org/:_authToken" "${NPM_TOKEN}"
-          pnpm run publish
-          pnpm config delete "//registry.npmjs.org/:_authToken"
+
+      # - name: "Continuous Deployment: publish to npm"
```

Comment:
Volgens mij is de reden dat we al 1.0 releases op NPM hebben staan, dat deze code niet was verwijderd.